app/assets/javascripts/editor.js

```diff
@@ -127,6 +127,37 @@ function create_editor() {
   }
 }
 
+function set_fullscreen(){
```

Comment:
The real `set_fullscreen` that is actually used is in `application.js`. This looks like remains from before the editor was disconnected from the fullscreen button. Same goes for the `exit_fullscreen `function below.